README.md

```diff
@@ -56,52 +56,55 @@ Before merging the latest changes from `develop` into a release branch:
+  * Use this configuration: [soliumrc].
+  * Make sure all changes pass the linter before opening a pull request.
 
 ### Directory Structure
 
-* All contract source code should be below the directory `./contracts`
-* All unit test and related code should be below the directory `./test`
-* All integration test and related code should be below the directory `./test_integration`
-* All build artifacts of a library project should be in the directory `./lib`
-* All documentation should be below the directory `./docs`
+* All contract source code should be below the directory `./contracts`.
+* All unit test and related code should be below the directory `./test`.
+* All integration test and related code should be below the directory `./test_integration`.
+* All build artifacts of a library project should be in the directory `./lib`.
+* All documentation should be below the directory `./docs`.
 
 ## JavaScript
 
-* Follow the [Airbnb style guide]
-* Any project with JavaScript code should have [eslint] installed
-  * Use this configuration: [eslintrc]
-  * Make sure all changes pass the linter before opening a pull request
+* Follow the [Airbnb style guide].
+* Any project with JavaScript code should have [eslint] installed.
+  * Use this configuration: [eslintrc].
+  * Make sure all changes pass the linter before opening a pull request.
 
 ### Directory Structure
 
-* All source code should be below the directory `./src`
-* All unit test and related code should be below the directory `./test`
-* All integration test and related code should be below the directory `./test_integration`
-* All build artifacts of a library project should be in the directory `./lib`
-* All documentation should be below the directory `./docs`
+* All source code should be below the directory `./src`.
+* All unit test and related code should be below the directory `./test`.
+* All integration test and related code should be below the directory `./test_integration`.
+* All build artifacts of a library project should be in the directory `./lib`.
+* All documentation should be below the directory `./docs`.
 
 ## `package.json`
 
 This section applies to Solidity and JavaScript projects alike.
 
-* `package.json` should include all required build steps in `scripts.prepack`
+* `package.json` should include all required build steps in `scripts.prepack`.
 * Provide at least the following `package.json` scripts:
   * `test`
   * `test:integration`
   * `build`
+  * `prepack`
+    * Should run all steps required before packing (publishing) the project (probably at least `npm run build`).
   * `lint`
-* Define `files` to minimize the size of the published package
-* `main` should point to `./lib/index.js`
+* Define `files` to minimize the size of the published package.
+* For library projects, `main` should point to `./lib/index.js`.
+
+### Packaged Projects
+
+⚠️ When you use a packaging tool like [webpack], make sure that `npm install` is part of the `prepack` script.
```

Comment:
@schemar: does this line apply to L97 (re: what is required of the `prepack` script)? Or no, because this here, with respect to using a packaging tool, is a still more specific case?

Reply:
Also, should it be `npm ci` instead? Per https://github.com/OpenST/mosaic.js/pull/169

Reply:
Yes and yes, thanks! I updated the PR accordingly.